app/controllers/your-psp/get.controller.test.js

```diff
@@ -64,4 +67,78 @@ describe('Your PSP GET controller', () => {
     expect(pageData.enableStripeOnboardingTaskList).to.equal(false)
     sinon.assert.calledWith(res.render, 'your-psp/index')
   })
+
+  it('should get stripe taskList and for your PSP', async () => {
```

Comment:
I would recommend changing to `should get stripe taskList when using Stripe`

Reply:
sounds good